static/app/views/settings/organizationIntegrations/repositoryProjectPathConfigRow.tsx

```diff
@@ -116,7 +116,7 @@ export const InputPathColumn = styled(Column)`
   grid-area: input-path;
 `;
 
-export const ButtonWrapper = styled(Column)`
+export const ButtonWrapper = styled('span')`
```

Comment:
note that `Column` is just:

```
const Column = styled('span')`
  overflow: hidden;
  overflow-wrap: break-word;
`;
```

so without hidden overflow, there is nothing left.